packages/fuel-indexer-tests/tests/e2e/graphql_api_postgres.rs

```diff
@@ -64,6 +73,10 @@ async fn test_can_return_query_response_with_all_fields_required_postgres() {
 #[actix_web::test]
 #[cfg(all(feature = "e2e", feature = "postgres"))]
 async fn test_can_return_query_response_with_nullable_fields_postgres() {
+    let fuel_node_handle = tokio::spawn(setup_test_fuel_node(
```

Comment:
I see pretty much every test that uses `setup_test_fuel_node` has this same snippet:


```rust
    let fuel_node_handle = tokio::spawn(setup_test_fuel_node(
        get_test_chain_config_path(),
        Some(get_test_contract_bin_path()),
    ));
```
Is there any way we can cut down on this boilerplate if we know that all tests will be using the same params? E.g.,:

```rust
    let fuel_node_handle = tokio::spawn(setup_test_fuel_node());
```
And if some non-test code is calling `setup_test_fuel_node` (e.g., `hello-world-test-node`) maybe we can wrap `setup_test_fuel_node` in something like `setup_example_fuel_node` which would call  `setup_test_fuel_node` under the hood?

Reply:
Yeah, using your suggestion in the last sentence is what I'll do.